goqtdrv5/cdrv.h

```diff
@@ -60,8 +63,9 @@ typedef uint8_t goBool;
 
 
 #define drvInvalid (void*)-1
+#define GO_STRING_POINTER_SIZE 0x10
```

Comment:
Right i overlooked that.. Ill take a look a little later, the
setStringArray isnt working as expected either so that needs to be fixed
On Mon, Sep 7, 2015 at 11:09 AM Utkan Güngördü <notifications@github.com>
wrote:

> In goqtdrv5/cdrv.h
> <https://github.com/salviati/go-qt5/pull/26#discussion_r38868275>:
>
> > @@ -60,8 +63,9 @@ typedef uint8_t goBool;
> >
> >
> >  #define drvInvalid (void*)-1
> > +#define GO_STRING_POINTER_SIZE 0x10
>
> This should be 8 bytes on i386 and arm (for reference, cgo provides types
> like GoString, intgo, ...).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/salviati/go-qt5/pull/26/files#r38868275>.
>